### DISCUSSION TITLE
Electrum not Runing on Raspiblitz v. 1.8.0

### DISCUSSION THREAD

**Discussion by thenewkid2022** [community/community#38099] [Other Feature Feedback, Questions, & Ideas]
Hi Community

Below is the Link for the Log-File from the Raspiblitz running on a Rasperypi.

[https://l.termbin.com/71j8]

I'm not a programmer so tried to find the error first by myself - but don't know where exactly the problem is.

It started out by manual installing LNDG for the Lightning Network. There seems to be an issue with the nginx serivce for the web server.
Why the Electrum Server is not running anymore is weird, it was working recently.

Thx in advance for the help

**Comment by ossanna16** [comment]
Hi @thenewkid2022,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Raspberry Pis. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
